From bf242dbaf09b78c7722239a583871499365fe79f Mon Sep 17 00:00:00 2001
From: Jintao Zhang <zhangjintao9020@gmail.com>
Date: Thu, 3 Jun 2021 10:57:54 +0800
Subject: [PATCH 1/1] remove containerd.io version limit

Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>
---
 rpm/SPECS/docker-ce.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm/SPECS/docker-ce.spec b/rpm/SPECS/docker-ce.spec
index 4bbd7f6f8c..05752362fe 100644
--- a/rpm/SPECS/docker-ce.spec
+++ b/rpm/SPECS/docker-ce.spec
@@ -23,7 +23,7 @@ Requires: libseccomp >= 2.3
 Requires: systemd
 Requires: iptables
 Requires: libcgroup
-Requires: containerd.io >= 1.4.1
+Requires: containerd.io
 Requires: tar
 Requires: xz
 
-- 
2.31.1

