From 235821e97facb84621ac4e68e4e74322d4dae077 Mon Sep 17 00:00:00 2001
From: Jintao Zhang <zhangjintao9020@gmail.com>
Date: Sun, 27 Sep 2020 13:01:42 +0800
Subject: [PATCH] remove containerd.io version limit

Signed-off-by: Jintao Zhang <zhangjintao9020@gmail.com>
---
 rpm/SPECS/docker-ce.spec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rpm/SPECS/docker-ce.spec b/rpm/SPECS/docker-ce.spec
index 6cb8729..5b7adf9 100644
--- a/rpm/SPECS/docker-ce.spec
+++ b/rpm/SPECS/docker-ce.spec
@@ -29,7 +29,7 @@ Requires: ( iptables or nftables )
 Requires: iptables
 %endif
 Requires: libcgroup
-Requires: containerd.io >= 1.3.0
+Requires: containerd.io
 Requires: tar
 Requires: xz
 
-- 
2.25.4

